app/apptesting/test_suite.go

```diff
@@ -43,6 +43,8 @@ import (
 
 type KeeperTestHelper struct {
 	suite.Suite
+	hasUsedAbci bool
+	withCaching bool // defaults to false
```

Comment:
Could you add comments around when we expect these to be toggled please?

Reply:
done!